Sources/AppcuesKit/Presentation/Debugger/FloatingView/FloatingView.swift

```diff
@@ -169,3 +211,39 @@ internal class FloatingView: UIView {
         return delegate != nil
     }
 }
+
+@available(iOS 13.0, *)
+private extension DebugMode {
+    var accessibilityLabel: String {
+        switch self {
+        case .debugger:
+            return "Appcues Debug Panel"
+        case .screenCapture:
+            return "Appcues Screen Capture"
+        }
+    }
+
+    var imageView: UIView {
+        switch self {
+        case .debugger:
+            let imageView = UIImageView(image: UIImage(asset: Asset.Image.debugIcon))
+            imageView.backgroundColor = .secondarySystemBackground
+            return imageView
+        case.screenCapture:
+            return FloatingImageView(image: UIImage(asset: Asset.Image.captureScreen), size: CGSize(width: 24.0, height: 24.0))
```

Comment:
My preference would be to update the asset to include the gradient for the sake of simplicity, but this certainly works.